### PR TITLE
feat(cloud): add pci deals route

### PR DESCRIPTION
**Pull request by marie-j**
⚠️ This relies on new 2API

### CODE REVIEW

File: src/api/cloud/cloud.aapi.service.js [hunk 1]
```diff
@@ -8,6 +8,11 @@ angular.module('ovh-api-services').service('OvhApiCloudAapi', ($resource, $cache
       serviceType: 'aapi',
       cache: queryCache,
     },
+    getDeals: {
```
Comment (FredericEspiau) [inline src/api/cloud/cloud.aapi.service.js]: Looks like an array, even though it's not 😅

File: src/api/cloud/cloud.aapi.service.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ angular.module('ovh-api-services').service('OvhApiCloudAapi', ($resource, $cache
...
+      method: 'GET',
+      url: '/public-cloud-deals',
```
Comment (antleblanc) [inline src/api/cloud/cloud.aapi.service.js]: To keep consistency across endpoint definition I would recommend you to keep endpoint prefix which is: `/cloud/instances/`

Comment (marie-j) [reply src/api/cloud/cloud.aapi.service.js]: I agree to keep `/cloud` but not `/instances` as it doesn't impact only instances.
Does `/cloud/deals` suits you ?

Comment (antleblanc) [reply src/api/cloud/cloud.aapi.service.js]: LGTM

**Pull request merged**